src/Contexts/WeatherContext.js

```diff
@@ -28,27 +28,28 @@ export const WeatherContextProvider = (props) => {
         getWeather()
         .then((result) => {
             setWeather(result);
-            //create miniweather content at the same time
-            const now = result.list[0];
-            console.log(now);
-            const returnable = {
-                temp: Math.round((now.main.temp-273.15 || currWeather.temp)* 10) / 10 + '° C',
-                feelsLike: Math.round((now.main.feels_like-273.15 || currWeather.feelsLike)* 10) / 10 + '° C',
-                wind: Math.round((now.wind.speed || currWeather.wind)* 10) / 10 + ' m/s',
-                humidity: Math.round((now.main.humidity || currWeather.humidity)* 10) / 10 + ' %',
-                weather: now.weather[0].main || currWeather.weather,
-                weatherDesc: now.weather[0].description || currWeather.weatherDesc,
-                iconNum: now.weather[0].icon || currWeather.iconNum,
-                cloudiness: now.clouds.all || now.cloudiness
-            };
-            setCurrWeather(returnable);
+            createCurrentWeather(result);
+            updateSixHours(result);
         });
     }
 
-    const updateSixHours = () => {
+    const createCurrentWeather = (weatherData) => {
+        const now = weatherData.list[0];
+        //refactor weather item into a form you want
+        const returnable = {
+            temp: Math.round((now.main.temp-273.15 || currWeather.temp)* 10) / 10 + '° C',
```

Comment:
can you add back feelslike, description of weather, and cloudiness back because I was using them for the weather page